bin/plugin/commands/performance.js

```diff
@@ -285,7 +285,7 @@ async function runPerformanceTests( branches, options ) {
 
 	log( '    >> Installing dependencies and building packages' );
 	await runShellScript(
-		'npm ci && node ./bin/packages/build.js',
+		'bash -c "source $HOME/.nvm/nvm.sh && nvm install && nvm use && npm ci && node ./bin/packages/build.js"',
```

Comment:
There's probably a better way to do this but it was hard to get nvm into the path here.

Reply:
AFAIR, calling `nvm install` also sets the installed version, so we don't need to call `nvm use`.